sys/linux/socket_netlink_route.txt

```diff
@@ -850,6 +878,58 @@ br_mdb_entry_sub_u [
 	in6_addr	ipv6_addr
 ]
 
+rtm_nh_policy_get_id [
+	NHA_ID	nlattr[NHA_ID, int32[0:2]]
+] [varlen]
+
+rtm_nh_policy_get_dump [
+	NHA_OIF		nlattr[NHA_OIF, ifindex]
+	NHA_GROUPS	nlattr[NHA_GROUPS, void]
+	NHA_MASTER	nlattr[NHA_MASTER, int32[0:2]]
+	NHA_FDB		nlattr[NHA_FDB, void]
+] [varlen]
```

Comment:
We can merge these two, and use it just as `rtm_nh_policy_get`.

We use them in the same way, thus, it doesn't bring anything.

It could've been useful if we further seperated them with their `netlink_msg.flag` to ensure `dump` gets the certain required flag. However, it seems overcomplicating. Also, the number of options for union here are not much, thus, it shouldn't affect the fuzzer efficiency by much.